kernel/src/main/scala/org/apache/toree/kernel/protocol/v5/interpreter/InterpreterActor.scala

```diff
@@ -96,5 +99,13 @@ class InterpreterActor(
             ex.getStackTrace.map(_.toString).toList)
           )
       } pipeTo sender
+    case (isCompleteRequest: IsCompleteRequest) =>
+      logger.debug(s"InterpreterActor requesting is complete code ${isCompleteRequest.code}")
+      (isCompleteTask ? isCompleteRequest) recover {
+        case ex: Throwable =>
+          logger.warn(s"Could determine completeness for code ${isCompleteRequest.code}: " +
```

Comment:
Looks like each interpreter should at least return an unknown status, so I think a warn is appropriate to bring to the surface that something is going wrong with the check. Furthermore, we log an error for the other message routes, so I think a warn is appropriate considering we are not bubbling up an exception.